scifight/models.py

```diff
@@ -1,6 +1,10 @@
 from django.db   import models
 from django.core import exceptions
+from django.utils import timezone
 
+SLUG_LENGTH = 20
+""" Maximum length of slug. This value must be large enough to display
+    the name of the team of tournament, given whitespace."""
```

Comment:
How about the following comment: «Maximum length of slug *fields*. This value must be large enough to hold a URL path component of reasonable length. No one is going to remember and read overly long addresses.»